### ISSUE TITLE
Loss get larger when lr decrease to 0.0001 or small, and it is obvious when the lr change 

### ISSUE THREAD

**Issue by trantorrepository** [auroua/InsightFace_TF#59]

**Comment by david-di** [comment]
You use 0.0001 as base learning rate? Or you means when your lr up to 0.0001, the loss get larger?
Can use share more details about this appearance? Maybe some training logs.
I'm very curious of this appearance, thank you.

**Comment by lovejing0306** [comment]
> You use 0.0001 as base learning rate? Or you means when your lr up to 0.0001, the loss get larger?
> Can use share more details about this appearance? Maybe some training logs.
> I'm very curious of this appearance, thank you.

I meet similarity problem, this is my train process:

epoch 0, total_step 880, total loss is 3.01 , inference loss is 1.10, weight deacy loss is 1.91, training accuracy is 0.812500, lr is 0.000300, time 124.273 samples/sec, total_time 0.514997
epoch 0, total_step 900, total loss is 2.87 , inference loss is 0.96, weight deacy loss is 1.91, training accuracy is 0.781250, lr is 0.000300, time 125.305 samples/sec, total_time 0.510756
epoch 0, total_step 920, total loss is 16637847666688.00 , inference loss is 1.30, weight deacy loss is 16637847666688.00, training accuracy is 0.796875, lr is 0.000300, time 125.331 samples/sec, total_time 0.510646
epoch 0, total_step 940, total loss is 20928329154560.00 , inference loss is 1.00, weight deacy loss is 20928329154560.00, training accuracy is 0.843750, lr is 0.000300, time 125.013 samples/sec, total_time 0.511949
epoch 0, total_step 960, total loss is 21482346381312.00 , inference loss is 0.81, weight deacy loss is 21482346381312.00, training accuracy is 0.843750, lr is 0.000300, time 124.458 samples/sec, total_time 0.514230
